apps/prairielearn/src/pages/enroll/enroll.html.ts

```diff
@@ -141,81 +85,14 @@ export function Enroll({
                                       type="button"
                                       class="btn btn-sm btn-danger"
                                       data-toggle="modal"
-                                      data-target="#removeModal${idx}"
+                                      data-target="#remove-course-modal"
+                                      data-course-instance-id="${course_instance.course_instance_id}"
+                                      data-course-instance-short-label="${course_instance.short_label}"
                                     >
                                       Remove course
                                     </button>
-                                    <div
-                                      class="modal fade"
-                                      id="removeModal${idx}"
-                                      tabindex="-1"
-                                      role="dialog"
-                                      aria-labelledby="removeModal${idx}Title"
-                                      aria-hidden="true"
-                                    >
-                                      <div
-                                        class="modal-dialog modal-dialog-centered"
```

Comment:
After approving I noticed this modal is using `modal-dialog-centered`, which is not supported by the modal component. This is not a big deal, as I don't think centering it on the screen is that important in this context, I just want to make sure this is taken into account.

Reply:
Yeah, I don't think this is a big deal.